src/node_buffer.cc

```diff
@@ -9,8 +9,28 @@
 #include "v8.h"
 
 #include <string.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <errno.h>
 #include <limits.h>
 
+#include <fstream>
+#include <string>
+#include <sstream>
+#include <iostream> // std::hex
+
+#ifdef _WIN32
+  #define __alignof__ __alignof
+  #define snprintf(buf, bufSize, format, arg) _snprintf_s(buf, bufSize, _TRUNCATE, format, arg)
```

Comment:
Also seems to be [completely unnecessary](https://github.com/nodejs/io.js/blob/2bb2f06b3e53ca44687afdaa9b833e0b6181ff76/src/node_internals.h#L65-L76). Whoops! Removing.